apps/measures/models.py

```diff
@@ -197,14 +201,73 @@ def facilitative_administration(self):
 
     @property
     def measures_pages(self):
-        measures_pages = MeasuresDetailPage.objects.live().descendant_of(self)
+        measures_pages = MeasuresDetailPage.objects.live()\
+            .descendant_of(self).order_by("title", "-first_published_at")
 
         return measures_pages
 
+    def _q_filter(self, filter_list, field_name):
+        filter_params = Q()
+        for element in filter_list:
+            kwargs = {'{0}__contains'.format(field_name): element}
```

Comment:
Do we need the 0?

Reply:
No, with one element, it's also readable without.